odoo/addons/base/res/res_country_view.xml

```diff
@@ -110,6 +110,9 @@
             <field name="model">res.country.state</field>
             <field name="arch" type="xml">
                 <form string="State">
+                    <div class="oe_button_box">
```

Comment:
No sorry, we already have this debate way too often in the code with hook methods, we do not wish to add empty attributes/code just because somebody may use it. It just makes things harder to follow and it is very plausible they will be removed in a refactoring without proper justification to add them.